x-pack/test/functional/services/ml/stack_management_jobs.ts

```diff
@@ -194,5 +198,99 @@ export function MachineLearningStackManagementJobsProvider(
+      } else {
+        await testSubjects.missingOrFail('~mlJobMgmtImportJobsFileReadErrorCallout');
+        await testSubjects.existOrFail('mlJobMgmtImportJobsFileRead');
+      }
+    },
+
+    async assertJobIdsExist(expectedJobIds: string[]) {
+      const inputs = await testSubjects.findAll('mlJobMgmtImportJobIdInput');
+      // const inputs = await subj.findAllByTagName('input');
+      const actualJobIds = await Promise.all(inputs.map((i) => i.getAttribute('value')));
+
+      expect(actualJobIds.sort()).to.eql(
+        expectedJobIds.sort(),
+        `Expected job ids to be '${JSON.stringify(expectedJobIds)}' (got '${JSON.stringify(
+          actualJobIds
+        )}')`
+      );
+    },
+
+    async assertCorrectTitle(jobCount: number, jobType: JobType) {
+      const dataTestSubj =
+        jobType === 'anomaly-detector'
+          ? 'mlJobMgmtImportJobsADTitle'
+          : 'mlJobMgmtImportJobsDFATitle';
+      const subj = await testSubjects.find(dataTestSubj);
+      const title = (await subj.parseDomContent()).html();
+
+      const jobTypeString =
+        jobType === 'anomaly-detector' ? 'anomaly detection' : 'data frame analytics';
+
+      const results = title.match(
+        /(\d) (anomaly detection|data frame analytics) job[s]? read from file$/
+      );
+      expect(results).to.not.eql(null, `Expected regex results to not be null`);
+      const foundCount = results![1];
+      const foundJobTypeString = results![2];
+      expect(foundCount).to.eql(
+        jobCount,
+        `Expected job count to be '${jobCount}' (got '${foundCount}')`
+      );
+      expect(foundJobTypeString).to.eql(
+        jobTypeString,
+        `Expected job count to be '${jobTypeString}' (got '${foundJobTypeString}')`
+      );
+    },
+
+    async assertJobIdsSkipped(expectedJobIds: string[]) {
+      const subj = await testSubjects.find('mlJobMgmtImportJobsCannotBeImportedCallout');
+      const skippedJobTitles = await subj.findAllByTagName('h5');
+      const actualJobIds = (
+        await Promise.all(skippedJobTitles.map((i) => i.parseDomContent()))
+      ).map((t) => t.html());
+
+      expect(actualJobIds.sort()).to.eql(
+        expectedJobIds.sort(),
+        `Expected job ids to be '${JSON.stringify(expectedJobIds)}' (got '${JSON.stringify(
+          actualJobIds
+        )}')`
+      );
+    },
+
+    async importJobs() {
+      await retry.tryForTime(5000, async () => {
+        await testSubjects.click('mlJobMgmtImportImportButton', 1000);
+      });
+      await retry.tryForTime(40000, async () => {
+        await testSubjects.missingOrFail('mlJobMgmtImportJobsFlyout');
+      });
+    },
+
+    async assertImportedJobIdsExist(expectedJobIds: string[]) {
+      await retry.tryForTime(40000, async () => {
+        await testSubjects.click('mlRefreshJobListButton', 1000);
+        await testSubjects.missingOrFail('mlJobMgmtImportJobsFlyout');
+      });
+    },
```

Comment:
Not sure if this method is needed. It seems to be validating something different than the method name suggests and it doesn't seem to be used.